eng/pipelines/libraries/base-job.yml

```diff
@@ -23,10 +23,10 @@ parameters:
 jobs:
   - template: /eng/common/templates/job/job.yml
     parameters:
-      ${{ if notIn(parameters.framework, 'allConfigurations', 'net472') }}:
+      ${{ if and(notIn(parameters.framework, 'allConfigurations', 'net472'), ne(parameters.isSourceBuild, true)) }}:
         displayName: ${{ format('Libraries {0} {1}{2} {3} {4}', parameters.displayName, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
         name: ${{ format('libraries_{0}_{1}{2}_{3}_{4}', parameters.name, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
-      ${{ if in(parameters.framework, 'allConfigurations', 'net472') }}:
+      ${{ if or(in(parameters.framework, 'allConfigurations', 'net472'), eq(parameters.isSourceBuild, true)) }}:
         displayName: ${{ format('Libraries {0} {1} {2} {3} {4}', parameters.displayName, parameters.osGroup, parameters.framework, parameters.archType, parameters.buildConfig) }}
         name: ${{ format('libraries_{0}_{1}_{2}{3}_{4}_{5}', parameters.name, parameters.framework, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
```

Comment:
This uses `parameters.framework` for the name, in this case it is empty. I think we should just add another entry for sourceBuild and customize the name and display name accordingly so that we can identify the build from the UI and in the logs.